tests/data/interpreter-groups.toml

```diff
@@ -2,7 +2,7 @@
 name = "igs"
 description = "Test interpreter group selection."
 
-[lift.scie-jump]
+[lift.scie_jump]
```

Comment:
N.B.: Before adding any tests, the existing tests failed here on the scie-jump mis-spell. This same silent (before) error was what bit @huonw in scie-pants recently.